go.mod

```diff
@@ -6,7 +6,7 @@ replace github.com/rkvst/go-rkvstcommon => ./
 
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.7.1
-	github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus v1.2.1
+	github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus v1.4.0
```

Comment:
is this version bump intentional ?

Reply:
yes - we dont get the CodeUnauthorizedAccees in errors.go otherwise